CHANGELOG.md

```diff
@@ -2,12 +2,57 @@
+- (Contrib) `awss3receiver`: Add support RFC3339 format for starttime and endtime ([#36787](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36787))
+- (Contrib) `fileexporter`: Add support for profiles signal. ([#35978](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35978))
+- (Contrib) `pkg/ottl`: GetXML Converter now supports selecting text, CDATA, and attribute (value) content. ([#36821](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36821))
+- (Contrib) `kafkaexporter, kafkareceiver`: Add a new mechanism "AWS_MSK_IAM_OAUTHBEARER" for kafka exporter and kafka receiver. This mechanism use the AWS MSK IAM SASL Signer for Go https://github.com/aws/aws-msk-iam-sasl-signer-go. ([#19747](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/19747))
+- (Contrib) `loadbalancingexporter`: Adds a an optional configuration to the k8s resolver which returns hostnames instead of IPs for headless services pointing at statefulsets ([#18412](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18412))
+- (Contrib) `mongodbatlasreceiver`: Adds additional metrics to the MongoDB Atlas receiver ([#36525](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36525))
+  Adds a number of new default disabled metrics to the MongoDB Atlas receiver. These metrics are:
+  - mongodbatlas.disk.partition.queue.depth
+  - mongodbatlas.disk.partition.throughput
+  - mongodbatlas.process.cache.ratio
+- (Contrib) `s3exporter`: Upgrading to adopt aws sdk v2 ([#36699](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36699))
+
+### 🧰 Bug fixes 🧰
+
+- (Core) `exporterhelper`: Fix memory leak at exporter shutdown ([#11401](https://github.com/open-telemetry/opentelemetry-collector/issues/11401))
+- (Core) `sharedcomponent`: Remove race-condition and cleanup locking ([#11819](https://github.com/open-telemetry/opentelemetry-collector/issues/11819))
+- (Contrib) `receiver/prometheusreceiver`: Metric adjuster no longer assumes that all metrics from a scrape come from the same resource ([#36477](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36477))
+- (Contrib) `googlecloudmonitoringreceiver`: Fix "no validation found" error if workload is running on Google Cloud Platform ([#36607](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36607))
+- (Contrib) `k8sattributesprocessor`: Override extracted k8s attributes if original value has been empty ([#36373](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36373))
+- (Contrib) `k8sattributesreceiver`: Log any errors encountered during kube client initialisation ([#35879](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35879))
```

Comment:
```suggestion
- (Contrib) `k8sattributesprocessor`: Log any errors encountered during kube client initialisation ([#35879](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35879))
```
This changelog was a typo in the upstream entry (not your fault), but we should fix it here